openwisp_monitoring/monitoring/migrations/0005_create_ping.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations
+from openwisp_monitoring.check.tasks import auto_create_ping
+
+from openwisp_controller.config.models import Device
```

Comment:
don't do this, use the migration framework to get the Device model, it's explained here: https://docs.djangoproject.com/en/3.0/topics/migrations/#data-migrations

See `apps.get_model`, there's an example with a related comment which explains why you have to use that instead of the concrete model.

Reply:
Thanks for pointing towards right resources. :smile: